CMakeLists.txt

```diff
@@ -13,6 +13,7 @@ set(CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/cmake")
 if(POLICY CMP0127)
     cmake_policy(SET CMP0127 OLD)
 endif()
+cmake_policy(SET CMP0074 NEW)
```

Comment:
Since we have explicitly set `cmake_minimum_required(VERSION 3.14)`, then CMP0074 (introduced in CMake 3.12) should already be set to `NEW`. Was there some error occurring without this explicit policy set?

Reply:
According to https://cmake.org/cmake/help/latest/policy/CMP0074.html:
> This policy was introduced in CMake version 3.12. CMake version 3.22.1 warns when the policy is not set and uses `OLD` behavior. Use the `cmake_policy()` command to set it to `OLD` or `NEW` explicitly.

Reply:
Yes, that is the default behavior. However, [`cmake_minimum_required()`](https://cmake.org/cmake/help/latest/command/cmake_minimum_required.html#policy-settings) calls `cmake_policy()` with the version specified in the `cmake_minimum_required()` call, which implicitly does `cmake_policy(SET CMP0074 NEW)`.